models/Metadata/Predefined.php

```diff
@@ -78,6 +78,11 @@ final class Predefined extends Model\AbstractModel
      */
     protected $language;
 
+    /**
+     * @var string
+     */
+    public $group;
```

Comment:
please change this to protected

Reply:
Fixed. (I actually developed this for Pimcore 6 and just copied the changes)